inst/tests/tests.Rraw

```diff
@@ -16496,18 +16508,18 @@ test(2125.02, capture.output(print(DT, trunc.cols=TRUE)),
        "102: 0 bbbbbbbbbbbbb ccccccccccccc",
        "1 variable(s) not shown: [d]"))
 test(2125.03, capture.output(print(DT, trunc.cols=TRUE, row.names=FALSE)),
-     c("    a             b             c",
-       "    0 bbbbbbbbbbbbb ccccccccccccc",
-       "    0 bbbbbbbbbbbbb ccccccccccccc",
-       "    0 bbbbbbbbbbbbb ccccccccccccc",
-       "    0 bbbbbbbbbbbbb ccccccccccccc",
-       "    0 bbbbbbbbbbbbb ccccccccccccc",
-       "---                              ",
-       "    0 bbbbbbbbbbbbb ccccccccccccc",
-       "    0 bbbbbbbbbbbbb ccccccccccccc",
-       "    0 bbbbbbbbbbbbb ccccccccccccc",
-       "    0 bbbbbbbbbbbbb ccccccccccccc",
-       "    0 bbbbbbbbbbbbb ccccccccccccc",
+     c("   a             b             c",
+       "   0 bbbbbbbbbbbbb ccccccccccccc",
+       "   0 bbbbbbbbbbbbb ccccccccccccc",
+       "   0 bbbbbbbbbbbbb ccccccccccccc",
+       "   0 bbbbbbbbbbbbb ccccccccccccc",
+       "   0 bbbbbbbbbbbbb ccccccccccccc",
+       " ---           ---           ---",
```

Comment:
WDYT about narrowing the column in this case? `toprint` is already small and type character, what about something like

```r
col_widths = pmax(nchar(names(toprint)), apply(toprint, 2L, function(x) max(nchar(x))))
n_dashes = pmin(col_widths, 3L)
dashes = substr(rep("---", ncol(toprint)), 1L, n_dashes)
```

Reply:
IMO, the current output looks the best. I'm just thinking in terms of situations where we have many columns with 1-3 character-long names, the different number of dashes may look a bit strange, whereas having the 3 dashes for every column is a bit more consistent.

Reply:
OK. It will be easy enough to change if requested later.